db/version_set.h

```diff
@@ -1136,6 +1137,21 @@ class VersionSet {
     }
     return min_log_num;
   }
+  uint64_t PreComputeMinLogNumberWithUnflushedData(
```

Comment:
Similar to the comments in line 1123 and 1124, a short explanation of this method can be helpful.